dataset-spec/dataset-spec.md

```diff
@@ -21,10 +21,9 @@ Implementations are encouraged, however, as good effort will be made to not chan
 | title       | string            | A short descriptive one-line title for the dataset.          |
 | description | string            | **REQUIRED.** Detailed multi-line description to fully explain the entity. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
 | keywords    | [string]          | List of keywords describing the dataset.                     |
-| version     | string            | Version of the dataset. [Semantic Versioning (SemVer)](https://semver.org/) SHOULD be followed. |
+| version     | string            | Version of the dataset.                                      |
```

Comment:
Why is semver being removed? Not against removing, but don't remember discussing it.

Reply:
That's feedback from within openEO (see my PR description), this does not originate from our telco. SemVer is very software centric and doesn't really always suit to dataset versioning.

Reply:
Cool, sorry, read the description quickly. Sounds reasonable to me.